doc/man7/RAND_DRBG.pod

```diff
@@ -265,9 +265,9 @@ from the trusted entropy sources.
 =back
 
 NOTE: Manual reseeding is *not allowed* in FIPS mode, because
-NIST SP-800-90A mandates that entropy *shall not* be provided by the
-consuming application, neither for instantiation, nor for reseeding.
-[NIST SP 800-90Ar1, Sections 9.1 and 9.2]. For that reason the B<randomness>
+[NIST SP 800-90A Rev. 1] mandates that entropy *shall not* be provided by
+the consuming application, neither for instantiation (Section 9.1), nor for
+reseeding (Section 9.2). For that reason the B<randomness>
```

Comment:
Note to @paulidale: It was my fault, I [was confused](https://github.com/openssl/openssl/pull/8722#issuecomment-491179796): The "neither, ..., nor ..." formulation was present twice. I kept the formulation but rearranged the citations slightly, in order to make its meaning more clear.